stdlib/math.jou

```diff
@@ -1,40 +1,107 @@
+# Math lib
+
+# Other functions
+# Returns the absolute value of x: |x|.
 declare abs(x: int) -> int
 declare llabs(x: long) -> long
 declare fabs(x: double) -> double
+# Returns x*y+z.
+declare fma(x: double, y: double, z: double) -> double
 
+# Rounding and remainder functions
+# Rounds x upward, returning the smallest integral value that is not less than x.
 declare ceil(x: double) -> double
+# Rounds x downward, returning the largest integral value that is not greater than x.
 declare floor(x: double) -> double
+# Returns the floating-point remainder of numer/denom (rounded towards zero):
+# fmod = numer - tquot * denom
+declare fmod(numer: double, denom: double) -> double
+# Returns the integral value that is nearest to x, with halfway cases rounded away from zero.
 declare round(x: double) -> double
+declare lround(x: double) -> long
+# Returns the floating-point remainder of numer/denom (rounded to nearest):
+# remainder = numer - rquot * denom
+declare remainder(numer: double, denom: double) -> double
 
+# Trigonometric functions
+# These functions use radians as the angle unit.
+# One radian is the angle of a circle slice with equal radius and arc length, about 57 degrees.
+# A full turn (360 degrees) is 2pi (about 6.28) radians.
+declare cos(x: double) -> double
+declare sin(x: double) -> double
+declare tan(x: double) -> double
+# The 'a' prefixed functions are inverse functions.
+# For example, asin is also known as arcsin and sin^-1.
 declare acos(x: double) -> double
 declare asin(x: double) -> double
 declare atan(x: double) -> double
+# Returns the angle of a point(x, y). Note the reversed order of the arguments.
 declare atan2(y: double, x: double) -> double
-declare cos(x: double) -> double
-declare sin(x: double) -> double
-declare tan(x: double) -> double
 
-declare fmod(x: double, y: double) -> double
-declare sqrt(x: double) -> double
-declare cbrt(x: double) -> double
+# Floating-point manipulation functions
+# Returns a value with the magnitude of x and the sign of y
+# Example:
+#   printf("%f", copysign(4, -1))
+# Output:
+#   -4.0
+declare copysign(x: double, y: double) -> double
+# Returns a quiet NaN (Not-A-Number) value of type .
+# The NaN values are used to identify undefined or non-representable values for floating-point elements, such as the square root of negative numbers or the result of 0/0.
+# The argument can be used by library implementations to distinguish different NaN values in a implementation-specific manner.
```

Comment:
The description is confusing, because it doesn't explain what a "library implementation" is, what is "a implementation-specific manner", and most importantly, it doesn't tell how to use the function.

```suggestion
# Use nan("") to get a quiet NaN (Not-A-Number) value.
# The argument must be an empty string.
```